Kickstarter-iOS/Views/Controllers/LandingPageViewController.swift

```diff
@@ -0,0 +1,228 @@
+import Library
+import Prelude
+import ReactiveSwift
+import UIKit
+
+private enum Layout {
+  enum Button {
+    static let height: CGFloat = 54
```

Comment:
Any reason we're not using our `Styles.minTouchSize.height` for buttons here?

Reply:
Oh good catch!